src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_dispatch_fails.cs

```diff
@@ -50,11 +60,24 @@ public Endpoint()
                 {
                     config.EnableFeature<TimeoutManager>();
                     config.UsePersistence<FakeTimeoutPersistence>();
-                    config.SendFailedMessagesTo(ErrorQueueForTimeoutErrors);
+                    config.SendFailedMessagesTo(Conventions.EndpointNamingConvention(typeof(Endpoint)));
                     config.RegisterComponents(c => c.ConfigureComponent<FakeTimeoutStorage>(DependencyLifecycle.SingleInstance));
+                    config.Pipeline.Register<BehaviorThatLogsControlMessageDelivery.Registration>();
+                    config.LimitMessageProcessingConcurrencyTo(1);
```

Comment:
I've reread the FLR implementation. You're right, this limitation isn't needed. I'll remove it.